services/Directory/data/FilterListViewUrl.json

```diff
@@ -15909,5 +15909,145 @@
     "id": 2813,
     "primariness": 3,
     "url": "https://cdn.statically.io/gh/JohnyP36/Personal-List/main/Personal%20List%20(uBo).txt"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2814,
+    "primariness": 1,
+    "segmentNumber": 1,
+    "url": "https://mypdns.org/my-privacy-dns/porn-records/-/raw/master/submit_here/adult.mypdns.cloud/wildcard.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2815,
+    "primariness": 2,
+    "segmentNumber": 1,
+    "url": "https://raw.githubusercontent.com/mypdns/porn-records/master/submit_here/adult.mypdns.cloud/wildcard.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2816,
+    "primariness": 3,
+    "segmentNumber": 1,
+    "url": "https://raw.githubusercontent.com/porn-records/porn-records/master/submit_here/adult.mypdns.cloud/wildcard.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2817,
+    "primariness": 4,
+    "segmentNumber": 1,
+    "url": "https://gitlab.com/MypDNS/porn-records/-/raw/master/submit_here/adult.mypdns.cloud/wildcard.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2818,
+    "primariness": 1,
+    "segmentNumber": 2,
+    "url": "https://mypdns.org/my-privacy-dns/porn-records/-/raw/master/submit_here/adult.mypdns.cloud/tld.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2819,
+    "primariness": 2,
+    "segmentNumber": 2,
+    "url": "https://raw.githubusercontent.com/mypdns/porn-records/master/submit_here/adult.mypdns.cloud/tld.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2820,
+    "primariness": 3,
+    "segmentNumber": 2,
+    "url": "https://raw.githubusercontent.com/porn-records/porn-records/master/submit_here/adult.mypdns.cloud/tld.list"
+  },
+  {
+    "filterListId": 2566,
+    "id": 2821,
+    "primariness": 4,
+    "segmentNumber": 2,
+    "url": "https://gitlab.com/MypDNS/porn-records/-/raw/master/submit_here/adult.mypdns.cloud/tld.list"
+  },
+  {
+    "filterListId": 2567,
+    "id": 2818,
```

Comment:
`id` values need to be unique here. this one is the same as the one up on line 15943

sorry this process is a bit painful. 😔

Reply:
Thanks for spotting the error, it clearly a mistake as I copy pasted the 4 entries a number of times